openssl_fips.yaml

```diff
@@ -0,0 +1,2 @@
+name: openssl-fips*
+url: https://builds.delivery.puppetlabs.net/openssl-fips/202409160/artifacts/
```

Comment:
From my understanding, this is a private/interal URL and means that this is not used to build our packages. Is OpenSSL (non-fips) included in puppet-runtime?

ref: https://github.com/OpenVoxProject/openvox-agent/blob/a5d0ab6fa1b2c315e003045c8473a75cdbb40b03/configs/components/openssl-fips.json

Reply:
As far as I know that's only required for fips packages and we don't build those right now.